fenix/app/src/main/res/xml/search_preferences_new.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
**nit:** Let's just this file to `search_settings_preferences` since it will align with the fragment toolbar name and it avoids having to rename this later on.

Reply:
implemented